File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.cpp
class Solution {
public:
    int arraySign(vector<int>& nums) {
        int n = nums.size();
        int neg = 0;
        int final = 0;
        for(int i = 0; i < n;i++){
            if(nums[i] == 0){
                return 0;
            }
            else if(nums[i] < 0){
                neg++;
            }
        } // end of for
        if(neg % 2 == 0){
            return 1;
        }else{
            return -1;
        }
    }
};

File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        if(nums.count(0) > 0): return 0
        neg = 0
        for num in nums:
            if num < 0:
                neg +=1
        if neg % 2 == 0:
            return 1
        else:
            return -1